melos.yaml

```diff
@@ -1,4 +1,4 @@
-name: amplify_flutter
+name: amplify_flutter_repository
```

Comment:
Why this change?

Reply:
This was a weird issue by melos where it would consider the melos config name as top level package and conflicts with the new amplify_flutter package.

Reply:
If you can would you mind raising a issue over at https://github.com/invertase/melos so I can take a look - doesn't seem right that this should be causing any conflicts but I could be wrong also 😅

Reply:
@Salakar https://github.com/invertase/melos/issues/47